tests/congruence_tests/test_query_batch.py

```diff
@@ -0,0 +1,195 @@
+        return client.query_batch_points(
+            collection_name=COLLECTION_NAME, requests=self.multivector_query_batch_text
+        )
+
+    def dense_query_batch_text(self, client: QdrantBase) -> models.QueryResponse:
+        return client.query_batch_points(
+            collection_name=COLLECTION_NAME, requests=self.dense_vector_query_batch_text
+        )
+
+    def dense_query_batch_image(self, client: QdrantBase) -> models.QueryResponse:
+        return client.query_batch_points(
+            collection_name=COLLECTION_NAME, requests=self.dense_vector_query_batch_image
+        )
+
+    def dense_query_batch_code(self, client: QdrantBase) -> models.QueryResponse:
+        return client.query_batch_points(
+            collection_name=COLLECTION_NAME, requests=self.dense_vector_query_batch_code
+        )
+
+    @staticmethod
+    def dense_query_batch_empty(client: QdrantBase) -> models.QueryResponse:
+        return client.query_batch_points(collection_name=COLLECTION_NAME, requests=[])
+
+
+# ---- TESTS  ---- #
+
+
+def test_sparse_query_batch():
+    fixture_points = generate_sparse_fixtures()
+
+    searcher = TestQueryBatchSearcher()
+
+    local_client = init_local()
+    init_client(local_client, fixture_points, sparse_vectors_config=sparse_vectors_config)
+
+    remote_client = init_remote()
+    init_client(remote_client, fixture_points, sparse_vectors_config=sparse_vectors_config)
+
+    compare_client_results(local_client, remote_client, searcher.sparse_query_batch_text)
+
+
+def test_multivec_query_batch():
+    fixture_points = generate_multivector_fixtures()
+
+    searcher = TestQueryBatchSearcher()
+
+    local_client = init_local()
+    init_client(local_client, fixture_points, vectors_config=multi_vector_config)
+
+    remote_client = init_remote()
+    init_client(remote_client, fixture_points, vectors_config=multi_vector_config)
+
+    compare_client_results(local_client, remote_client, searcher.multivec_query_batch_text)
+
+
+def test_dense_query():
+    fixture_points = generate_fixtures()
+
+    searcher = TestQueryBatchSearcher()
+
+    local_client = init_local()
+    init_client(local_client, fixture_points)
+
+    remote_client = init_remote()
+    init_client(remote_client, fixture_points)
+
+    compare_client_results(local_client, remote_client, searcher.dense_query_batch_text)
+    compare_client_results(local_client, remote_client, searcher.dense_query_batch_image)
+    compare_client_results(local_client, remote_client, searcher.dense_query_batch_code)
+
+
+def test_dense_query_empty_batch():
+    fixture_points = generate_fixtures()
+
+    searcher = TestQueryBatchSearcher()
+
+    local_client = init_local()
+    init_client(local_client, fixture_points)
+
+    remote_client = init_remote()
+    init_client(remote_client, fixture_points)
+
+    compare_client_results(local_client, remote_client, searcher.dense_query_batch_empty)
```

Comment:
this assertion could go in the previous test `test_dense_query` to avoid creating a new collection